### ISSUE TITLE
VB -> C#: Duplicate Namespace

### ISSUE THREAD

**Issue by vfle** [icsharpcode/CodeConverter#201]
Please prefix the issue title with either "C# -> VB: " or "VB -> C#: " if relevant

### Input code
```vbnet
Namespace Child
```

### Erroneous output
```C#
namespace Root.Namespace.Root.Namespace.Child
```

### Expected output
```C#
namespace Root.Namespace.Child
```

### Details
Product in use: VS extension

Version in use: 6.1

The project is a class library with a root namespace project property set. At the example above it would be Root.Namespace .
I am not sure why the second root namespace appears.

It is an easy work around on the produced code to replace the duplicates, just reporting in case something more serious is hidden.

**Comment by GrahamTheCoder** [comment]
Thanks for the report. I suspect it's either a bug in the general `QualifyNode`, or that's working fine, but then something else re-qualifies it later.

**Comment by knodel12** [comment]
@GrahamTheCoder Wondering why you removed the RootNamespace logic.  It seems that all VB files that previously had no namespace now get created in C# without a namespace, regardless of the project root namespace.  Is this intentional?

**Comment by GrahamTheCoder** [comment]
Ah ok yes. In the case where the namespace is missing, we need to add it, but we don't need to add on to existing ones. I'll take a closer look.

**Comment by GrahamTheCoder** [comment]
I've pushed the bit to hopefully avoid the duplicate name. I'll push something else shortly to qualify namespaces better since it's in the same area of code.

**Comment by GrahamTheCoder** [comment]
See repro on #318

**Comment by GrahamTheCoder** [comment]
Fixed in 2483369355807050379e2385a4f22d16eb3437f7
